public/src/components/lab/LastBuildInfo.tsx

```diff
@@ -50,8 +50,11 @@ export class LastBuildInfo extends React.Component<ILastBuildInfo, ILastBuildInf
         if (delivered >= deadline) {
             classString = "past-deadline";
         }
-
         return <div className={classString}>{formatDate(delivered)}</div>;
     }
 
+    private formatTime(executionTime: number): number {
```

Comment:
We should just do this conversion on the server side; we already do a conversion (Line 61, `ci/results.go`):
```go
			ExecTime:  execTime.Milliseconds(),
```
Could be changed to:
```go
			ExecTime:  execTime.Seconds(),
```

Reply:
Should we also change `ExecTime` type to float64 then?